README.md

```diff
@@ -44,7 +44,7 @@ Please contribute. We are primarily interested in the following:
 
 After installing, please see the following areas to learn more:
 
-* [.NET IoT 101 (Jan 2020)](https://channel9.msdn.com/Series/IoT-101) - An introduction series how to create .NET IoT applications with a Raspberry Pi.
+* [.NET IoT 101 (Jan 2020)](https://channel9.msdn.com/Series/IoT-101) - An introduction series how to create .NET IoT applications on a Raspberry Pi.
```

Comment:
lol, I meant the on after series. See my code suggestion to better understand where did I meant the on to be added.

Reply:
SMH 😄